custom_components/elegoo_printer/elegoo_sdcp/models/printer.py

```
@@ -2,7 +2,12 @@
 
 import json
 
+from custom_components.elegoo_printer.elegoo_sdcp.models.print_history_detail import (
+    PrintHistoryDetail,
+)
+
 from .attributes import PrinterAttributes
+from .print_history_detail import PrintHistoryDetail
```

Ruff (0.8.2)

Redefinition of unused `PrintHistoryDetail` from line 6

(F811)